### PR TITLE
DRT-23 - Fix JAWS accessibility issue when navigating through list items

### PR DESCRIPTION
**Pull request by nachofregeiro**
We introduced some changes to the **SimpleSelect** and **Input** in order to follow the following specification: https://www.w3.org/TR/wai-aria-1.1/#combobox to make it work with JAWS.

The markup structure looks like this now (new stuff highlighted in yellow):

image

We also changed the **FocusPlaceholder** component not to be readonly to enable focus on that input and make the navigation work with JAWS when using a non searchable select. We updated the css width to 0px not to show the blinking cursor when the input is focused.

### CODE REVIEW

File: src/select/Input.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ const Input = ({
       id={id}
       aria-label={ariaProps['aria-label'] || undefined}
       aria-labelledby={ariaProps['aria-labelledby'] || undefined}
+      aria-multiline={ariaProps['aria-multiline'] || undefined}
```
Comment (vicky-holcomb) [inline src/select/Input.js:19]: Why this addition?  I don't see where it would ever be set to true.

Comment (nachofregeiro) [reply src/select/Input.js:19]: We are setting that prop in SimpleSelect line 683. 'aria-multiline': 'false' and role: 'textbox' are not required to work with JAWS but are recommended properties in the structure.

We can eventually get rid of them.

**Review by matt-leyrat — APPROVED** [review]

File: src/select/SimpleSelect.js [hunk 2]
```diff
@@ -686,7 +686,7 @@ class SimpleSelect extends Component {
       inputValue: this.props.inputValue,
       onBlur: this.onInputBlur,
       onChange: this.onInputChange,
-      onFocus: this.onInputFocus
+      onFocus: this.onInputFocus,
```
Comment (vicky-holcomb) [inline src/select/SimpleSelect.js:689]: nit: remove trailing comma

Comment (sthomas1618) [reply src/select/SimpleSelect.js:689]: agreed, please remove and we're good to merge.

**Review by sthomas1618 — APPROVED** [review]

**Pull request merged**